### DISCUSSION TITLE
Python Tetris does not deploy...

### DISCUSSION THREAD

**Discussion by mariavarg** [vercel/community#4571] [Help]
Why my Python Tetris does not deploy on Vercel??? It's running great locally... I am not using a framework... I thought Vercel can handle Python... My game is hosted on GitHub, but deploys on Vercel!!!

**Comment by amyegan** [comment]
Vercel does have a Python Runtime for Serverless Functions. 🙂  You can find more information about working with it in the docs: https://vercel.com/docs/functions/serverless-functions/runtimes/python

There are also some Python starter templates available: https://vercel.com/templates/python
